xml/installation-installation-ses_integration.xml

```diff
@@ -99,101 +114,32 @@
  radosgw_urls:
      - http://10.84.56.7:80/swift/v1
      - http://10.84.56.8:80/swift/v1 </screen>
-  <para>
-   After you have run the <literal>openstack.integrate</literal> runner, copy
-   the yaml into the <filename>ses_config.yml</filename> file on the deployer
-   node. Then edit the <filename>settings.yml</filename> file to enable &ses;
-   integration to run and update all of the &cloud; service configuration
-   files. The <filename>settings.yml</filename> file resides in the
-   <filename>~/openstack/my_cloud/config/ses</filename> directory. Open the
-   <filename>settings.yml</filename> file for editing, uncomment the
-   <literal>ses_config_path:</literal> parameter, and specify the location on
-   the deployer host containing the <filename>ses_config.yml</filename> file.
-  </para>
-   <para>
-    If you are integrating with &ses; and do not want to store &o_comp; images
-    in &ceph;, the following setting is required:
-   </para>
-   <para>
-    Edit <filename>settings.yml</filename> and change the line
-    <literal>ses_nova_set_images_type: True</literal>
-    to <literal>ses_nova_set_images_type: False</literal>
-   </para>
-   <procedure>
-    <step>
      <para>
-      Commit your configuration to your local git repo:
+      After you have run the <literal>openstack.integrate</literal> runner,
+      copy the yaml output into the new <filename>ses_config.yml</filename>
+      file, and save this file in the path specified in the
+      <filename>settings.yml</filename> file on the deployer node. In this
+      case, the file <filename>ses_config.yml</filename> must be saved in the
+      <filename>/var/lib/ardana/openstack/my_cloud/config/ses/</filename>
+      directory on the deployer node.
+     </para>
+
+     <para>
+      For &ses;/&ceph; deployments that have version older than 5.5, the
+      following applies. For &ceph;, it is necessary to create pools and users
+      to allow the &productname; services to use the &ses;/&ceph;
+      cluster. Pools and users must be created for &o_blockstore;,
+      &o_blockstore; backup, &o_comp; and &o_img;. Both the &o_blockstore; and
+      &o_comp; services must have the same user, as &o_blockstore; needs access
+      to create objects that &o_comp; uses.  Instructions for creating and
+      managing pools, users and keyrings is covered in the &ses; documentation
+      under <link
+      xlink:href="https://www.suse.com/documentation/suse-enterprise-storage-5/book_storage_admin/data/storage_cephx_keymgmt.html">Key
```

Comment:
Do note the old URL here. Please fix to: https://documentation.suse.com/en-us/ses/5.5/single-html/ses-admin/#storage-cephx-keymgmt